src/Components/Main.js

```diff
@@ -29,7 +28,8 @@ const Main = () => {
 
   };
   return (
-    <div className="flex justify-center items-center mt-12">
+    <div>
```

Comment:
Any particular reason why an extra div is added here ?

Reply:
Was thinking of adding a login/logout/signup button here. when the user logs in,only then he will be able to see his projects, else he'll just see the contents of the page or maybe some default page stating "signup to see your contents".